build.gradle

```diff
@@ -95,6 +95,8 @@ dependencies {
     compile fileTree(dir: 'lib', includes: ['*.jar'])
 
     compile 'com.google.guava:guava:19.0'
+    
+    compile 'com.airhacks:afterburner.fx:1.6.3'
```

Comment:
When adding a new library, please also add it to [external-libraries.txt](https://github.com/JabRef/jabref/blob/master/external-libraries.txt).